azure-login/README.md

```diff
@@ -0,0 +1,40 @@
+# GitHub Action for the Azure Login
+
+The GitHub Action for [Azure](https://azure.microsoft.com/) Login wraps the Azure CLI's `az login`, allowing for Actions to log into Azure.
+
+Because `$HOME` is persisted across Actions, the `az login` command will save this information on the filesystem, allowing other Actions to reuse the context.
+
+
+## Usage
+
+```
+
+ action "Azure Login" {
+  uses = "Azure/github-actions/login@master"
```

Comment:
Correct action name is azure-login

Reply:
@dikhakha . Same as above. Please update the name

Reply:
Done #36